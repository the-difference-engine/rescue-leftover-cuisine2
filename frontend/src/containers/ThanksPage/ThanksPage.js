import React from 'react';
import { Helmet } from 'react-helmet';
import { AuthPage } from '../../components/AuthItems/AuthItems';
import ThankYouCard from '../../components/ThankYouCard/ThankYouCard';

const ThanksPage = () => (
  <AuthPage>
    <Helmet>
      <title>Thanks!</title>
    </Helmet>
    <ThankYouCard />
  </AuthPage>
);

export default ThanksPage;
