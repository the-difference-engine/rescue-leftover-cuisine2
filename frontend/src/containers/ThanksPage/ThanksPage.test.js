import React from 'react';
import renderer from 'react-test-renderer';

import ThanksPage from './ThanksPage';

it('renders correctly', () => {
  const component = renderer
    .create(<ThanksPage />)
    .toJSON();

  expect(component).toMatchSnapshot();
});
