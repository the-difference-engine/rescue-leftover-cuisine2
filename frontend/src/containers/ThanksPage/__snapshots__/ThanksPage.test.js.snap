// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="auth-page container-fluid"
>
  <div
    className="row"
  >
    <div
      className="container-fluid text-center"
    >
      <a
        href="/"
      >
        <img
          alt="RLC Logo"
          className="logoCenter"
          src="RLC_logo.png"
        />
      </a>
    </div>
  </div>
  <div
    className="auth-page-content"
  >
    <div
      className="auth-cards-wrapper d-flex flex-row justify-content-center"
    >
      <div
        className="auth-card"
      >
        <h3
          className="auth-card-header"
        >
          <img
            alt="checkmark"
            className="thanks-check"
            src="checkmark-icon.png"
          />
          <br />
          Thank you for joining our community!
        </h3>
        <div
          className="thanks-welcome-text row"
        >
          Welcome to Rescuing Leftover Cuisine Recipes
        </div>
        <div
          className="row"
        >
          Check your email for instructions on how to verify your account.
        </div>
      </div>
    </div>
  </div>
  <div
    className="auth-footer row"
  >
    <div
      className="container-fluid containerSection"
    >
      <div
        className="footer"
      >
        <div
          className="row"
        >
          <div
            className="col-md-4"
          />
          <div
            className="col-md-4 company"
          >
            <span>
              Â©2018 Rescuing Leftover Cuisine
            </span>
          </div>
          <div
            className="col-md-4 iconSection"
          >
            <span
              className="float-right"
            >
              <a
                href="https://www.youtube.com/channel/UCJKK9jkEIHt8j17cHPHCPQw"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-youtube fa-2x iconStyle"
                />
              </a>
              <a
                href="https://www.facebook.com/RescuingLeftoverCuisine"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-facebook-f fa-2x iconStyle"
                />
              </a>
              <a
                href="https://twitter.com/rescuingcuisine"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-twitter fa-2x iconStyle"
                />
              </a>
              <a
                href="https://www.instagram.com/rescuingleftovercuisine"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-instagram fa-2x iconStyle"
                />
              </a>
              <a
                href="https://www.linkedin.com/company/rescuing-leftover-cuisine"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-linkedin-in fa-2x iconStyle"
                />
              </a>
              <a
                href="mailto:info@resuingleftovercuisine.org"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fas fa-envelope fa-2x iconStyle"
                />
              </a>
            </span>
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col-md-4"
          />
          <div
            className="col-md-4 designBySection"
          >
            <p>
              <span>
                Designed by 
              </span>
              <span
                className="designByCo"
              >
                Fluid Designs. 
              </span>
              App icons by
              <a
                href="https://icons8.com/"
                rel="noopener noreferrer"
                target="_blank"
              >
                 icons8.
              </a>
            </p>
          </div>
          <div
            className="col-md-4"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
